controllers/chat/moveToAnon.js

```diff
@@ -46,6 +62,7 @@ const moveToAnon = async (req, res) => {
   const oldChannelDetail = await client.queryChannels(filter, sort);
   //req.userId always anon since the token are using anon token
   const mySignedId = await UsersFunction.findSignedUserId(User, req.userId);
+  ////////////////////////
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line seems to be a placeholder or debugging artifact and should probably be removed.

```diff
-  ////////////////////////
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->